pcsx2/IopBios.cpp

```diff
@@ -559,7 +514,7 @@ namespace R3000A
 			}
 		}
 
-		bool is_host(const std::string path)
+		bool is_host(const std::string_view& path)
```

Comment:
`std::string_view` should be passed by value.

Reply:
`cdvdLoadElf` also takes it by reference, want me to change that too while I'm at it?